doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -43,142 +44,27 @@ Some Key Differences
        sequence is found using a[0].
 
    * - MATLAB®'s scripting language was created for doing linear algebra.
-       The syntax for basic matrix operations is nice and clean, but the API
-       for adding GUIs and making full-fledged applications is more or less
+       MATLAB's syntax for some array manipulations is more compact than
+       NumPy's.The syntax for basic matrix operations is nice and clean, but the
+       API for adding GUIs and making full-fledged applications is more or less
```

Comment:
Missing space after the full stop. Also, the first sentence and first half of the second sentence would be better combined.

Reply:
Thanks for catching that. I could go either way with combining the sentences. My preference is shorter sentences, but I'll combine and let's take another look.

Reply:
Looks good now, thanks.